doc/classes/AnimationTreePlayer.xml

```diff
@@ -68,6 +70,7 @@
 			<argument index="2" name="enable" type="bool">
 			</argument>
 			<description>
+				If [code]enable[/code] is [code]true[code], the animation node [code]id[/code] does not pass animation data to the node at relative [NodePath] [code]path[/code].
```

Comment:
Comment No. 3
Well it's not quite like that.. It just can filter out bones from the processing (so given bones will not be animated):
![peek 2017-12-08 11-22](https://user-images.githubusercontent.com/6129594/33761752-6e9052f2-dc0a-11e7-8b71-5df49c671c5e.gif)

Reply:
But if you deactivate a non-leaf node (a node that HAS children), do the children also stop animating? That's what I was communicating. I can add that the node in question also stops animating though.

Reply:
No children will not stop animating, only the disabled bone will not animate. It off-course might have eventual impact on how animation of children looks, but the children will animate normally in such case

Reply:
I would further modify this part: `off animations for the node at [code]path[/code]`
Actually it's a path related to the path that you can see in animation tracks of animationPlayer (it's a path to the property) (in case of skeletons each bone is represented as skeleton property) 
![peek 2017-12-08 16-18](https://user-images.githubusercontent.com/6129594/33771978-6385d71e-dc33-11e7-89ce-c72da650f4ad.gif)

Reply:
What I mainly meant in previous comment, is that skeleton don't have any 'sub nodes' or something (bones are not nodes). Plus also the fact that you don't need to have skeleton to use this, it's basically a way to disable animation tracks.

Reply:
How do you have the tree structure in the filter list if they are just the names of animation tracks in AnimationPlayer then? Can you nest AnimationPlayer animations within one another? That would be news to me...

Reply:
OH WAIT, I'm a dummy. Gotcha, relative path from the AnimationPlayer to the node and it's property that is being modified by the track.

Reply:
Yes, and the tree with bones that you could see on previous gif is probably an effect of property (bone) name parsing by node filters window.